### PR TITLE
Fix/3684 Translations and icons

### PR DESCRIPTION
**Pull request by asf-algoo**
#3684

## Checkpoints

**For developers**

- [X] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [X] Automated tests NOT RELEVANT

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Review by gbonaspetti — APPROVED** [review]

**Comment by grignards** [comment]
Is this:
image
the right icon for creating a space? It seems strange to me (same as created member?).

**Comment by grignards** [comment]
Also the tool-tip in join space is still not translated:
image

File: frontend/i18next.scanner/pt/translation.json [hunk 1]
```diff
@@ -286,7 +291,7 @@
   "{{author}} granted you access to {{space}}": "{{author}} concedeu-lhe acesso a {{space}}",
   "{{author}} rejected your access to {{space}}": "{{author}} rejeitou-lhe o acesso a {{space}}",
   "{{author}} granted access to {{space}} for {{user}}": "{{author}} concedeu acesso a  {{space}} para {{user}}",
-  "{{author}} rejected access to {{space}} for {{user}}": "{{author}} a rejecté l'accès à {{space}} para {{user}}",
+  "{{author}} rejected access to {{space}} for {{user}}": "{{author}} a rejeté l'accès à {{space}} para {{user}}",
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/pt/translation.json:294]:
```suggestion
  "{{author}} rejected access to {{space}} for {{user}}": "{{author}} rejeitou acesso a {{space}} para {{user}}",
```

File: frontend/i18next.scanner/pt/translation.json [hunk 2]
```diff
@@ -286,7 +291,7 @@
   "{{author}} granted you access to {{space}}": "{{author}} concedeu-lhe acesso a {{space}}",
   "{{author}} rejected your access to {{space}}": "{{author}} rejeitou-lhe o acesso a {{space}}",
   "{{author}} granted access to {{space}} for {{user}}": "{{author}} concedeu acesso a  {{space}} para {{user}}",
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/pt/translation.json:293]:
```suggestion
  "{{author}} granted access to {{space}} for {{user}}": "{{author}} concedeu acesso a {{space}} para {{user}}",
```

File: functionnal_tests/cypress/integration/join_workspace/join_workspace_page.spec.js [hunk 3]
```diff
@@ -54,4 +54,17 @@ describe('Join space page', () => {
     })
   })
 
+  describe('The icon tooltip', () => {
+    it("is translated", () => {
```
Comment (gbonaspetti) [inline functionnal_tests/cypress/integration/join_workspace/join_workspace_page.spec.js:58]:
```suggestion
    it('should be translated', () => {
```

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

**Review by gbonaspetti — APPROVED** [review]

**Comment by gbonaspetti** [comment]
This problems are fixed, but I found two new ones during tests, please, can you fix these translations too? 
- "Rejeiar pedido" to "Rejeitar pedido" at `frontend_app_workspace_advanced/i18next.scanner/pt/translation.json`
- and
Screenshot from 2020-10-30 14-42-18

**Comment by gbonaspetti** [comment]
Tested on branch

**Pull request merged**